### ISSUE TITLE
Crash after beating Woodfall

### ISSUE THREAD

**Issue by CowboyJoseph64** [leoetlino/project-restoration#219]
**Describe the bug**
After beating Woodfall Temple, the game will crash after zooming into the entrance on the back.

**To Reproduce**
Play The Game Normally. You have to play from at least the start of the second cycle, and go all the way to the end of the temple. I didn't do anything out of the ordinary I don't think, and it's happened both times I've tried. I got all the stray fairies, is that significant?

**Screenshots**
![image](https://user-images.githubusercontent.com/25892341/157750652-7c00da6c-611c-4dab-a437-cb78856da0f0.png)
This is rather useless as it's too blurry, but just in case you can gleam something from it.

**Additional context**
I tried a third time to get a better picture of that crash log, and it didn't happen. On the third time, I saved just before the boss so that I didn't have to start over again.

I use a Japanese model of the New 2DS XL, although I hacked it to have a US bios. I'm playing the American version of the game, and I'm using v1.1 of the patch, as 1.0 and 1.01 both crashed on boot.

**Comment by leoetlino** [comment]
Yeah, unfortunately that picture is too blurry. The address is almost completely unreadable (except the last two digits, "b4"?) and the error code is even worse.
